Demo/EarlGreyExample/EarlGreyExampleSwiftTests/BridgingHeader.h

```diff
@@ -0,0 +1,20 @@
+//
```

Comment:
Could you delete this file? I shouldn't have added it. This will be picked up with the gem.